tests/data/group_system/group_network/group_network-kernel/group_network_host_and_router_parameters/rule_sysctl_net_ipv4_conf_default_secure_redirects/correct_value.pass.sh

```diff
@@ -7,3 +7,5 @@ if grep -q "^net.ipv4.conf.default.secure_redirects" /etc/sysctl.conf; then
 else
 	echo "net.ipv4.conf.default.secure_redirects = 0" >> /etc/sysctl.conf
 fi
+
```

Comment:
@ggbecker I think ospp profile should be added in this and other scenarios because `C2S` profile isn't in RHEL8 and sysctl rules aren't part of RHEL8 `cui`.

Reply:
I've substituted older profiles with OSPP profile. This work in RHEL7 and should work with RHEL8 when new OSPP(CCC) profile gets in place.